StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/STPAPIClient+PaymentSheet.swift

```diff
@@ -13,19 +13,53 @@ import Foundation
 extension STPAPIClient {
     typealias STPIntentCompletionBlock = ((Result<Intent, Error>) -> Void)
 
-    func retrieveElementsSession(
-        paymentIntentClientSecret: String
-    ) async throws -> (STPPaymentIntent, STPElementsSession) {
-        let parameters: [String: Any] = [
-            "client_secret": paymentIntentClientSecret,
-            "type": "payment_intent",
-            "expand": ["payment_method_preference.payment_intent.payment_method"],
+    func makeElementsSessionsParams(mode: PaymentSheet.InitializationMode, configuration: PaymentSheet.Configuration) -> [String: Any] {
```

Comment:
super nit: should we just inject the `configuration.externalPaymentMethodConfiguration?.externalPaymentMethods` instead of the whole configuration object?